mafiasi/base/models.py

```diff
@@ -114,7 +114,7 @@ class LdapUser(LdapModel):
         'last_name': LdapAttr('sn'),
         'email': LdapAttr('mail'),
         'password': LdapAttr('userPassword'),
-        'owncloud_quota': LdapAttr('ownCloudQuota')
+        'nextcloud_quota': LdapAttr('ownCloudQuota')
```

Comment:
Hrm, maybe we should migrate our LDAP schemas to be able to change the LDAP attribute name as well?

Reply:
Yes, that would be better, but I don't know how